plugins/ftdidmx/FtdiDmxPlugin.cpp

```diff
@@ -44,14 +49,6 @@ const char FtdiDmxPlugin::PLUGIN_PREFIX[] = "ftdidmx";
  * Ownership of the FtdiDmxDevice is transfered to us here.
  */
 void FtdiDmxPlugin::AddDevice(FtdiDmxDevice *device) {
-  // Check if device is working before adding
```

Comment:
Gotcha. I wonder if its worth doing a basic can I connect check here. Or if you could check for and delete devices with no ports during StartHook() or similar. Thoughts @nomis52